presto-spi/src/main/java/io/prestosql/spi/type/RealType.java

```diff
@@ -72,6 +74,12 @@ public int compareTo(Block leftBlock, int leftPosition, Block rightBlock, int ri
         return Float.compare(leftValue, rightValue);
     }
 
+    @Override
+    public Optional<Range> getRange()
+    {
+        return Optional.of(new Range((long) Float.floatToRawIntBits(-Float.MAX_VALUE), (long) Float.floatToRawIntBits(Float.MAX_VALUE)));
```

Comment:
static import `floatToRawIntBits`

Please add a comment why the range does not include `POSITIVE_INFINITY`, `NEGATIVE_INFINITY`.